ivy/functional/backends/torch/activations.py

```diff
@@ -132,6 +134,6 @@ def mish(x: torch.Tensor, /, *, out: Optional[torch.Tensor] = None) -> torch.Ten
     backend_version,
 )
 def hardswish(
```

Comment:
GitHub won't let me make the suggestion because the decorator hasn't been edited, but you should have a `jax_like` function for `hardswish` which supports complex numbers and then you should be able to remove `"complex"` from the `unsupported_dtypes` decorator